web-animations-1/Overview.bs

```diff
@@ -1930,11 +1966,18 @@ non-normative description is also provided:
 
 :   <a lt="idle play state">idle</a>
 ::  The <a>current time</a> of the animation is <a>unresolved</a> and
+    the <a>start time</a> of the animation is <a>unresolved</a> and
     there are no pending tasks.
     In this state the animation has no effect.
 :   <a lt="running play state">running</a>
 ::  The animation has a resolved <a>current time</a> that changes on each
-    <a>animation frame</a> (provided the [=playback rate=] is not zero).
+    <a>animation frame</a> (provided the [=playback rate=] is not zero
+    and the <a>timeline</a> is [=monotonically increasing=]). If the
+    <a>timeline</a> is not [=monotonically increasing=], the
+    <a>current time</a> may not change on each <a>animation frame</a> or
+    is unresolved if the <a>timeline</a> is
+    <a lt="inactive timeline">inactive</a>. In the latter case
+    the animation has no effect.
```

Comment:
Perhaps this could be simplified to something like...

> The animation has a resolved current time that changes on each animation frame (provided the playback rate is not zero and the animation has an associated timeline that is active and monotonically increasing).

Reply:
Changed per your suggestion. However I still feel that we are not explaining behavior of animations attached to active/inactive finite timelines in terms of current time update and having effect.

Reply:
Yes, that's true. But I think this is non-normative text so perhaps it doesn't need to describe everything?